publication-commons/src/main/java/no/unit/nva/publication/service/impl/ResourceService.java

```diff
@@ -594,7 +608,10 @@ private void setStatusOnNewPublication(UserInstance userInstance, Publication fr
     }
 
     private List<Entity> refreshAndMigrate(List<Entity> dataEntries, CristinUnitsUtil cristinUnitsUtil) {
-        return dataEntries.stream().map(attempt(dataEntry -> migrate(dataEntry, cristinUnitsUtil))).map(Try::orElseThrow).toList();
+        return dataEntries.stream()
```

Comment:
Do we assume them to be distinct? I'd check the docs whether they are actually distinct.